extension/src/setup/autoInstall.ts

```diff
@@ -64,7 +81,12 @@ const showInstallProgress = (
     try {
       await Toast.runCommandAndIncrementProgress(
         async () => {
-          await installPackages(root, pythonBinPath, 'dvc')
+          await installPackages(
```

Comment:
I think some duplication could be removed here: 

```
    try {
      await Toast.runCommandAndIncrementProgress(
        async () => {
          await installPackages(
            root,
            pythonBinPath,
            ...getProcessGlobalArgs(isGlobalEnv),
            'dvc'
          )
          return 'DVC Installed'
        },
        progress,
        75
      )
      return Toast.delayProgressClosing()
    } catch (error: unknown) {
      return Toast.reportProgressError(error, progress)
    }
```
(especially with the upgrade command being basically the same with a different return value)

Reply:
Cleaned up some of the duplication!